operator/src/main/java/oracle/kubernetes/operator/rest/model/ConversionResponse.java

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator.rest.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import com.google.gson.annotations.Expose;
+
+public class ConversionResponse {
+
+  @Expose
+  private String uid;
+  @Expose
+  private List<Object> convertedObjects = new ArrayList<>();
+  @Expose
+  private Result result;
+
+  public String getUid() {
+    return uid;
+  }
+
+  public void setUid(String uid) {
+    this.uid = uid;
+  }
+
+  public ConversionResponse uid(String uid) {
+    this.uid = uid;
+    return this;
+  }
+
+  public java.lang.Object getConvertedObjects() {
+    return convertedObjects;
+  }
+
+  public void setConvertedObjects(List<Object> convertedObjects) {
+    this.convertedObjects = convertedObjects;
+  }
+
+  public ConversionResponse convertedObjects(List<Object> convertedObjects) {
+    this.convertedObjects = convertedObjects;
+    return this;
+  }
+
+  public Result getResult() {
+    return result;
+  }
+
+  public void setResult(Result result) {
+    this.result = result;
+  }
+
+  public ConversionResponse result(Result result) {
+    this.result = result;
+    return this;
+  }
+
+  @Override
+  public String toString() {
+    return "ConversionResponse{"
+            + "uid='" + uid + '\''
+            + ", convertedObjects=" + convertedObjects
+            + ", result=" + result
+            + '}';
+  }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    ConversionResponse that = (ConversionResponse) o;
+    return Objects.equals(uid, that.uid)
+            && Objects.equals(convertedObjects, that.convertedObjects)
+            && Objects.equals(result, that.result);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(uid, convertedObjects, result);
```

Comment:
Minor Nit: I'm not familiar with REST programming - but it seems odd that the request's and response's hash and equals methods depend on values that can change via 'setters'. The usual practice is to make all such values final and set them in a constructor (to eliminate the chance of mutations after putting such objects in a hash map).

Reply:
Not sure if I'm understanding this correctly. Most of the model classes (e.g. ProbeTuning or ServerPod etc.) has hash and equals method that depend on values that can change via 'setters'. In fluent style of coding, we don't set values using constructor but have methods such as below to construct the object. 
```
    new ProbeTuning()
        .initialDelaySeconds(initialDelay)
        .timeoutSeconds(timeout)
        .periodSeconds(period);
```

Reply:
ok.